pkg/ccl/backupccl/backup_processor.go

```diff
@@ -91,6 +92,13 @@ var (
 		"send each export request with a verbose tracing span",
 		util.ConstantWithMetamorphicTestBool("export_request_verbose_tracing", false),
 	)
+
+	flushTracingAggregatorFrequency = settings.RegisterDurationSetting(
+		settings.TenantWritable,
+		"bulkio.backup.tracing_aggregator_flush_after",
+		"frequency at which backup tracing aggregator stats are flushed",
+		time.Second*1,
+	)
```

Comment:
This seems pretty aggressive combined with the 5 second hard-coded timer in the aggregator.  In the text we might want to make it clear that this is the rate at which they are flushed from the processors, not the rate at which they are flushed to storage.

Reply:
Whoops this was only for proving to myself it worked. Neither of these values should be in the seconds in my opinion. I'll fix this up!

Reply:
I'm not convinced what I have is ideal right now. Both the per-processor and coordinator flush intervals are governed by the same cluster setting that defaults to 1 minute. I did consider splitting it out into two cluster settings but it feels like overkill. if we are pushing more aggregator stats through the flow we should be flushing them to disk also more frequently.

In an ideal world I'd have this configurable per job, but cluster settings will have to do for now. Let me know what you think?

Reply:
I'm tempted to just say every processor should send every 15s, without a setting, and then have the setting only apply to the flushing in the coordinator (which as I mentioned in slack, I'm tempted to remove later in favor of isProfiling bool on the job record)